app/models/merits_assessment.rb

```diff
@@ -1,3 +1,22 @@
 class MeritsAssessment < ApplicationRecord
   belongs_to :legal_aid_application
+
+  enum(
+    success_prospect: {
+      better: '50% or better'.freeze,
+      marginal: 'Marginal 45 - 50%'.freeze,
+      borderline: 'borderline'.freeze,
+      uncertain: 'uncertain'.freeze,
+      poor: 'poor'.freeze
+    },
+    _prefix: true
+  )
+
+  def list_success_prospects
+    MeritsAssessment.success_prospects.drop(1).map { |k| k }
+  end
+
+  def first_success_prospect
+    MeritsAssessment.success_prospects.keys.first
+  end
```

Comment:
I have a little problem with the naming of methods in this Model. They imply that what is significant is the position of elements in the enum, and do not convey the intention of the class. I would suggest this:
```ruby
class MeritsAssessment < ApplicationRecord
  belongs_to :legal_aid_application

  enum(
    success_prospect: {
      likely: '50% or better'.freeze,
      marginal: 'Marginal 45 - 50%'.freeze,
      borderline: 'borderline'.freeze,
      uncertain: 'uncertain'.freeze,
      poor: 'poor'.freeze
    },
    _prefix: true
  )

  def prospect_likely_to_succeed
    :likely
  end

  def prospects_unlikely_to_succeed
    @prospects_unlikely_to_succeed ||= self.class.success_prospects.except(prospect_likely_to_succed.to_s)
  end
end
```
Obviously that will require a rewrite to replace the calls to `list_success_prospects` and `first_success_prospect`, and the use of `better` elsewhere in the code.

Reply:
Sorry - Looking at this again. The methods are specific to the class level object - not the instance. Therefore `prospects_unlikely_to_succeed` and `prospect_likely_to_succeed` should be class methods. Otherwise `prospects_unlikely_to_succeed` will be unnecessarily redetermined each time an instance is created, when it only needs to be done once. So:
```ruby
class MeritsAssessment < ApplicationRecord
  belongs_to :legal_aid_application

  # defining before enum so it can be used in the enum
  def self.prospect_likely_to_succeed
    :likely
  end

  enum(
    success_prospect: {
      prospect_likely_to_succeed => '50% or better'.freeze,
      marginal: 'Marginal 45 - 50%'.freeze,
      borderline: 'borderline'.freeze,
      uncertain: 'uncertain'.freeze,
      poor: 'poor'.freeze
    },
    _prefix: true
  )

  def self.prospects_unlikely_to_succeed
    @prospects_unlikely_to_succeed ||= success_prospects.except(prospect_likely_to_succed.to_s)
  end
end
```
It also allows `prospect_likely_to_succeed` to be used in the enum construction - DRYing things up a little more.